types/react-native-sensor-manager/index.d.ts

```diff
@@ -4,7 +4,7 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
-/// <reference types="react-native" />
+/// <reference types="react-native/v0.70" />
```

Comment:
I don't think the old version usage should be set here, since TS itself uses it for imports, and the local user won't have the same structure.

Reply:
I updated this due to this error: https://gist.github.com/lunaleaps/a184026d2c6cbeaa90f41f0c03f580f8
which doesn't seem to reference the `tsconfig.json` config for resolving `react-native` to the 0.70 version

Reply:
Seems like the issue is reported here: https://github.com/microsoft/TypeScript/issues/33206

Reply:
Removing it doesn't seem to impact the tests, `npm run react-native-sensor-manager` or `npm run test-all`, I think from the documentation triple-slash references usage should be dropped in favor of imports.. which the other DT packages seem to do

Reply:
I think the `tsconfig.json` should resolve `react-native` to the 0.70 instead. That way the publisher also ensures `react-native-sensor-manager` declares a dependency on 0.70 types since it's clearly not compatible. 

Or fix the `react-native-sensor-manager` types to be compatible with latest.

Reply:
Yea, the `tsconfig.json` paths are ignored. So I'm removing the triple-slash reference